use-cases/virgo/README.md

```diff
@@ -59,3 +59,40 @@ mlflow ui --backend-store-uri mllogs/mlflow
 # In background 
 mlflow ui --backend-store-uri mllogs/mlflow > /dev/null 2>&1 &
 ```
+
+## Running scaling tests
+Scaling tests have been integrated into the eurac usecase to provide timing of experiments run and ths show the power of distributed model training and itwinai. Refer to the following files `runall.sh , scaling-test.sh, torch_dist_final_scaling.py`.
+
+Launch the scaling test:
+
+```bash
+bash scaling-test.sh
```

Comment:
Why "bash" and not just "./scaling-test.sh"?

Reply:
Does that make a difference? I think both would work, right?